main.go

```diff
@@ -99,22 +98,21 @@ func main() {
 	username := args.Username
 	password := args.Password
 	
-	if args.UsernameFILE != "" && args.PasswordFILE != "" {
-		contentUser, err := ioutil.ReadFile(args.UsernameFILE)
+	if args.UsernameFile != "" && args.PasswordFile != "" {
+		contentUser, err := os.ReadFile(args.UsernameFile)
```

Comment:
See if you can use https://github.com/alexflint/go-arg/issues/202#issuecomment-1295881062

Reply:
@amir20 How to handle user and password differently?

Reply:
My knowledge of go is minimal.

Reply:
Not sure I understand. In that comment, it basically does what you did but `UnmarshalText` does reading from file. A little bit cleaner.

Reply:
I agree. But I still don't understand how to do it(

Reply:
Oh I see. I won't be able to get on laptop until Monday. But basically you want to do exactly what you did in pr. But read the file from UnmarshalText instead. That way you don't read the file in main().

Reply:
I almost got it. But I'm getting this error: You can't use args.Password (variable of type *string) as a type string in a struct literal  

```
if args.Username == nil && args.UsernameFile != nil {
        args.Username = &args.UsernameFile.Value
}
```

```
config := web.Config{
                Addr:     args.Addr,
                Base:     args.Base,
                Version:  version,
                TailSize: args.TailSize,
                Username: args.Username,
                Password: args.Password,
        }
```